.agents/skills/wpilib-sim/scripts/run_shot_map_sim.py

```
@@ -0,0 +1,395 @@
+        str(repo / ".agents/skills/wpilog-decode/scripts/read_wpilog_values.py"),
+        "--repo",
+        str(repo),
+        "--log",
+        str(log),
+        "--json",
+        "--samples",
+        "20",
+    ]
+    for topic in topics:
+        command.extend(["--topic", topic])
+    result = subprocess.run(command, cwd=repo, check=True, text=True, capture_output=True)
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)